app/javascript/components/visualization/ClusterControls.js

```diff
@@ -138,13 +147,13 @@ export default function ClusterControls({studyAccession,
   }, [studyAccession, preloadedAnnotationList])
 
   return (
-    <div>
+    <div className="cluster-controls">
       <div className="form-group">
         <label>Load cluster</label>
         <Select options={clusterOptions}
           value={{ label: renderParams.cluster, value: renderParams.cluster }}
           onChange={ cluster => setRenderParams({
-            annotation: getDefaultAnnotationForCluster(annotationList, cluster.name, renderParams.annotation),
+            annotation: annotationKeyProperties(getDefaultAnnotationForCluster(annotationList, cluster.name, renderParams.annotation)),
```

Comment:
Also, please assign something like `defaultAnnot` above to avoid this too-dense line:

```suggestion
            annotation: getAnnotKeys(getDefaultAnnotationForCluster(annotationList, cluster.name, renderParams.annotation)),
```